### PR TITLE
Applying styles

### LINKED ISSUE #33: As a user I want my application to be styled so that my webapp is more visually appealing
### Acceptance criteria
**Given**
* User has loaded the main page of the webapp

**When**
* User sees the map view of Europe

**Then**
* User observes that the application is styled with a blue theme

**Happy case**
- The entire application is styled

**Bad case**
- Only some views in the application is styled
- The application has no styling

### TASKS
- [x] (Rúni) Outsource the styling
- [x] (Camilla - #40) When receiving the outsourced styling - make sure it works with our application
  - [x] Check all button-placements and move those that overlap 
  - [x] Style the `/about/` page.
     - Maybe take inspiration from the gradient in `header`
- [x] Update footer text from *lorem ipsum* to actual
  - [x] Investigate font for the `github-logo-text`
image
- [x] Remove/Change gradient color in `header` - what do we want here? **`start over?`**?
- [x] Cleanup 

#### Result
  - [x] Add a `go back` button to the `Result` view
  - [x] On `Result`, change `eventName` to hold a link (`<a>`-tag) to ticketmaster. (`eventURL`)

#### DatePicker
- [x] Handle Date Ranges 
```md
https://github.com/ Hacker0x01 /react-datepicker/issues/584
``` 
(remove spaces, thx github)
- [x] `See events`-button
- [x] ~~`Today`-button (Maybe not that important, as we start selector on `new Date()`)~~

*Meget af koden der mangler kan hives ind fra tidligere branches-
Der er på nuværende master en fuld funktionel `back`-button på `Result` (med ekstra funktionalitet)
& de to **datepicker** knapper findes også.
Det eneste 'nye' er nok date ranges & styling af `about` samt af ovenstående knapper*

### PR DESCRIPTION
**Pull request by Castau**
Fixes #33
Styling in this project has been outsourced as specified by the assignment and is being implemented by [Anatolii Borkhovych](https://github.com/aborkhovych)

### When approving, please merge and deploy frontend 

After Anatolii was done @Castau changed the following:
- Hardcoded city has been removed
- Datepicker is a single calendar and has a button to "select"
  - There is a minor annoyance with the range, don't know if you want that fixed (and if I can)
- About view has been updated to match rest of styling
- Header has been fixed
  - Click a country is always a link to front view
  - Logo has been changed
- ALL City buttons have been checked and moved to avoid overlap
- OutputCountry has been changed
- Eventlist now has links and hover effect
- Footer has been fixed

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ function App() {
     const [startDate, setStartDate] = useState(new Date());
     const [endDate, setEndDate] = useState(new Date());
     // Date states end
-    const [city, setCity] = useState(""); 
-    const [country, setCountry] = useState("");
+    const [city, setCity] = useState("Copenhagen"); 
+    const [country, setCountry] = useState("Denmark");
```
Comment (Runi-VN) [inline src/App.js]: This should be *reverted* before we merge :)  -- it was used to provide test data for the `Result` component.

**Review by Runi-VN — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (Runi-VN) [inline src/App.js]: This should be *reverted* before we merge :)  -- it was used to provide test data for the `Result` component.
🤔 very precise 😂 
image

**Review by Runi-VN — CHANGES_REQUESTED** [review]

**Comment by Runi-VN** [comment]
# For reference 
*Before* vs. *after*
Concept-files given to @aborkhovych can be found [here](https://i.imgur.com/BBM8hc7.png).

## Before
#### `App.js` or Map view (Frontpage)
image
#### Zoomed in on country 
image
#### `Result.jsx` **event view**
image
#### `Result.jsx` **weather view**
image

## After
#### `App.js` or Map view (Frontpage)
image

#### Zoomed in on country 
**_Not available atm_**

#### `Result.jsx` **event view**
image

#### `Result.jsx` **weather view**
image

**Review by MrHardcode — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (Runi-VN) [reply src/App.js]: Fixed https://github.com/Hold-Krykke/3SemProject_FrontEnd/pull/40/commits/5c945c7918d0da54914977e1a0fa56edae47777a

**Review by Runi-VN — APPROVED** [review]
Added `target="blank"` to all URLs heading off-site (In the `About`, `Footer` & `Result` components) so that they open in a new tab.

I am very happy with the result.

**Pull request merged**